src/pages/user-guide.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import Layout, { PageBody, SectionContent } from "../components/layout"
+import SEO from "../components/seo"
+
+export default function UserGuide() {
+  return (
+    <Layout>
```

Comment:
This `<Layout>` component is duplicating the header

Reply:
Thanks for the catch Wille! Removed